components/cosmetic_filters/resources/data/content_cosmetic.ts

```diff
@@ -9,6 +9,8 @@
 // - for cosmetic filters work with CSS and stylesheet. That work itself
 //   could call the script several times.
 
+import { applyCompiledSelector, compileProceduralSelector } from './procedural_filters'
```

Comment:
@antonok-edm @bridiver 
This results in significant increasing the JS inject size to every frame.

I hope you estimated the extra loading delay somehow. Could you share the numbers?

Reply:
@atuchin-m to make sure I understand, there's an increase in load times? Does that happen on every page, or only pages with an active filter rule that uses a procedural filter? And does it happen in Standard mode, or also Aggressive mode?

Reply:
@ShivanKaul
It add an overhead to any page load (we always inject the script if Shields aren't down)
The script size changed in 2x times: 
<img width="1409" alt="content_cosmetic_ts_size" src="https://github.com/user-attachments/assets/8e116c68-2986-4db1-ae99-b99b47688b7d">

I don't see any major changes in high-level metrics on the dashboard now, but every such change (+10k JS inject) makes Brave harder to compete again Chrome.